doc/additional_commands.rst

```diff
@@ -51,7 +51,20 @@ Universum in non-CI mode has the following differences from default mode:
 Poll chosen VCS for updates
 ---------------------------
 
-.. TODO: Add description
+This mode was created for CI systems, not having polling feature at all or exact functionality, listed below.
```

Comment:
"This mode was created for CI systems that do not have polling feature at all or are not able to check each change individually."